cmd/devices-api/api.go

```diff
@@ -193,6 +193,7 @@ func startWebAPI(logger zerolog.Logger, settings *config.Settings, pdb db.Store,
 	v1Auth.Post("/user/devices", userDeviceController.RegisterDeviceForUser)
 
 	v1Auth.Get("/integrations", userDeviceController.GetIntegrations)
+	v1Auth.Get("/integration/:tokenID", userDeviceController.GetIntegrationNFTMetadata)
```

Comment:
You don’t want to require authentication for this, it’s for public use.

Could we place this next to the other NFT endpoints?